todo_app/app.py

```diff
@@ -10,10 +11,21 @@
 @app.route('/')
 def index():
     items = get_items()
-    return render_template("index.html", items=items)
+    return render_template("index.html", items=items, ItemStatus=ItemStatus)
 
 @app.route('/item', methods=["POST"])
 def create_item():
     item_title = request.form.get('item_title')
-    add_item(item_title)
+    item_desc = request.form.get('item_desc')
+    add_item(item_title, item_desc)
+    return redirect('/')
+
+@app.route('/item/<id>/complete')
```

Comment:
Generally better to stick to POST requests for anything that makes changes.

More semantic plus you won't accidentally make changes by reloading a page in your browser/clicking on links.

Though it does mean your HTML needs to be _slightly_ more complicated - a `<form>` with a submit button instead of just an `<a>`.